proto/frequenz/api/common/v1/metrics/metrics.proto

```diff
@@ -172,6 +172,49 @@ enum Metric {
   METRIC_SENSOR_IRRADIANCE = 166;
 }
 
+// Enumerated categories of connections from which metrics can be obtained for a
+// given electrical component or sensor.
+enum MetricConnectionCategory {
+  // Default value.
+  METRIC_CONNECTION_CATEGORY_UNSPECIFIED = 0;
+
+  // Represents a connection to a battery.
+  METRIC_CONNECTION_CATEGORY_BATTERY = 1;
+
+  // Represents a connection to a PV array.
+  METRIC_CONNECTION_CATEGORY_PV_ARRAY = 2;
+
+  // Represents a connection to a temperature sensor measuring the ambient
+  // temperature.
+  METRIC_CONNECTION_CATEGORY_AIR = 10;
+
+  // Represents a connection to a temperature sensor measuring the
+  // temperature of a cabinet or enclosure.
+  METRIC_CONNECTION_CATEGORY_CABINET = 11;
+
+  // Represents a connection to a temperature sensor measuring the
+  // temperature of a heatsink or cooling element.
+  // Represents a connection to a temperature sensor measuring the
+  //
+  // temperature of a transformer.
+  METRIC_CONNECTION_CATEGORY_TRANSFORMER = 13;
+}
+
+// Represents a connection from which a metric was obtained.
+message MetricConnection {
+  // The category of the connection from which the metric was obtained.
+  MetricConnectionCategory code = 1;
```

Comment:
This also sounds inconsistent. I would either name `MetricConnectionCode` or name the field `category`.